tests/test_ransac.cc

```diff
@@ -137,4 +137,61 @@ TEST(test_outlier, test_ransac_groups) {
   EXPECT_LE(result.inliers.size(), indexer.size());
 }
 
+inline bool never_accept_candidates(const std::vector<FoldName> &) {
+  return false;
+}
+
+RansacConfig get_reasonable_ransac_config() {
```

Comment:
nit: I love the joke, but I'm unsure how it balances with professionalism.

Reply:
Heh yeah there was no joke intended. changed the NAN to a 0. .